pom.xml

```diff
@@ -1,72 +1,109 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  
-  <groupId>ProjectKorraRPG</groupId>
-  <artifactId>ProjectKorraRPG</artifactId>
-  <version>1.0.0</version>
-  <packaging>jar</packaging>
-  <name>ProjectKorraRPG</name>
-  
-  <repositories>
-  	<repository>
-  	  <id>spigot-repo</id>
-      <url>https://hub.spigotmc.org/nexus/content/repositories/snapshots/</url>
-  	</repository>
-  </repositories>
-  
-  <dependencies>
-  	<dependency>
-	  <groupId>org.spigotmc</groupId>
-	  <artifactId>spigot-api</artifactId>
-	  <version>1.14.3-R0.1-SNAPSHOT</version>
-	  <scope>provided</scope>
-	</dependency>
-	
-	<dependency>
-	  <groupId>com.projectkorra</groupId>
-	  <artifactId>projectkorra</artifactId>
-	  <version>1.8.8</version>
-	  <scope>provided</scope>
-	</dependency>
-  </dependencies>
-  
-  <build>
-    <defaultGoal>clean package install</defaultGoal>
-    <finalName>${project.name}-${project.version}</finalName>
-    <sourceDirectory>${project.basedir}/src/</sourceDirectory>
-    <resources>
-      <resource>
-        <targetPath>.</targetPath>
-        <filtering>true</filtering>
-        <directory>${project.basedir}/src/</directory>
-        <includes>
-          <include>*.yml</include>
-        </includes>
-      </resource>
-    </resources>
-    <plugins>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-compiler-plugin</artifactId>
-        <version>3.1</version>
-        <configuration>
-          <source>1.8</source>
-          <target>1.8</target>
-        </configuration>
-      </plugin>
-	  <plugin>
-	    <artifactId>maven-shade-plugin</artifactId>
-		<version>2.3</version>
-		<executions>
-		  <execution>
-		    <phase>package</phase>
-			<goals>
-			  <goal>shade</goal>
-			</goals>
-		  </execution>
-		</executions>
-	    <configuration />
-	  </plugin>
-    </plugins>
-  </build>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>ProjectKorraRPG</groupId>
+    <artifactId>ProjectKorraRPG</artifactId>
+    <version>1.0.0</version>
+    <packaging>jar</packaging>
+
+    <name>ProjectKorraRPG</name>
+
+    <properties>
+        <author>ProjectKorra</author>
+        <main.class>com.projectkorra.rpg.ProjectKorraRPG</main.class>
```

Comment:
Why does this have a main class?

Main classes are what java runs when you try execute a jar. Plugins don't have them :)